test/nightly/azureResourceMongoDB.test.ts

```diff
@@ -83,4 +91,10 @@ suite('MongoDB action', async function (this: ISuiteCallbackContext): Promise<vo
         const connectionString: string = await vscode.env.clipboard.readText();
         return await connectToMongoClient(connectionString, appendExtensionUserAgent());
     }
+
+    async function doesMongoDatabaseExist(): Promise<IDatabaseInfo | undefined> {
```

Comment:
Make databaseName1 a parameter that can be passed in that way we can reuse it for any other Database checks we do.  Eventually we can probably make it so the API is also a parameter so we can use it for SQL, Table, and Gremlin as well.